### PR TITLE
Docs: added section in slices-pattern explaining how to affect multiple stores in one function

### PR DESCRIPTION
**Pull request by Rec0iL99**
## Related Issues

fixes #1365 

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Review by Aliemeka — COMMENTED** [review]
Can you include in this doc an implementation for returning all the stores back to default. In other words like  a `clear` or `reset` functionality.

**Comment by Rec0iL99** [comment]
> Can you include in this doc an implementation for returning all the stores back to default. In other words like a `clear` or `reset` functionality.

The `reset` functionality can be found in the `How to reset state` doc. I can maybe provide a link to the doc from here. Will this be required @dai-shi @chrisk-7777?

File: docs/guides/slices-pattern.md [hunk 1]
```diff
@@ -62,6 +62,37 @@ function App() {
 export default App
 ```
 
+### Changing multiple stores together
+
+You can affect multiple stores together in a single function itself.
```
Comment (chrisk-7777) [inline docs/guides/slices-pattern.md:67]: What do you think about:

"You can update multiple stores, at the same time, in a single function."

 - "affect" may hold a slightly different meaning (for example, like that in useEffect or a side effect)
 - "itself" feels redundant at the end of the sentence. 

In terms of the heading, do you think something succinct like:

"Updating multiple stores"

Thoughts @Rec0iL99 ?

Comment (Rec0iL99) [reply docs/guides/slices-pattern.md:67]: After thinking about it, yes I agree.

**Review by chrisk-7777 — APPROVED** [review]
Thanks for actioning that feedback. The rest looks good 👍

File: docs/guides/slices-pattern.md [hunk 2]
```diff
@@ -62,6 +62,37 @@ function App() {
...
+You can update multiple stores, at the same time, in a single function.
+
+```js
+import { createBearSlice } from './bearSlice'
+import { createFishSlice } from './fishSlice'
+
+export const createBearFishSlice = (set) => ({
+  addBearAndFish: () => {
+    createBearSlice(set).addBear()
+    createFishSlice(set).addFish()
```
Comment (dai-shi) [inline docs/guides/slices-pattern.md:76]: Interesting. I haven't thought about this pattern.

**Review by dai-shi — APPROVED** [review]
Thanks!

**Pull request merged**